src/candidates/profile/chat/shared/navigation/ChatNavigation.vue

```diff
@@ -69,7 +68,28 @@ export default {
   methods: {
     ...mapActions({
       selectContact: 'chat/selectContact'
-    })
+    }),
+    filterItemsMatchingSearch (list) {
+      if (!this.searchText) {
+        return list;
+      }
+
+      const includesIgnoreCase = (str, suffix) =>
+        str.toLowerCase().includes(suffix.toLowerCase());
```

Comment:
I was initially considering extracting this function as a utility into the core module, but ultimately found it more readable to inline it here.